packages/mirrorful/cli/init.ts

```diff
@@ -31,7 +32,7 @@ export async function init({
   await makeDir('.mirrorful')
 
   const port = 5050 // don't hard code this
-
+  await openBrowser({ url: 'http://localhost', port })
```

Comment:
Awesome! I think this line should be moved to the end. I think it should be on line 107 instead, after ensuring there are no errors

Reply:
If we move it all the way to the bottom it seems to not open the browser.

Reply:
Oh interesting, makes sense